compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmValueGen.java

```diff
@@ -309,14 +300,12 @@ private void createInstantiateMethod(ClassWriter cw, BRecordType recordType,
         }
         mv.visitInsn(POP);
 
```

Comment:
These newlines are not added by me. I think they are added by the original author to group the code lines logically to improve clarity since the method is long. Hence I think it is better to keep them and remove only extra blank lines.